content/neuvector-prime/azure/contents.lr

```diff
@@ -349,6 +333,17 @@ AKS cluster already has the extension. Uninstall the extension as it suggested i
    !["Uninstall extension"](./uninstall.png)
 
 
+### To check Kubernetes Service Extension version is latest:
+
+With Auto upgrade minor version" enabled, te deployment should always be on the latest version. To see the 3 most recent published versions,
```

Comment:
s/te/the

Reply:
With Auto upgrade minor version" enabled, the deployment should always be on the latest version. To see the 3 most recent published versions, Execute the following command in Cloud Shell

Reply:
done